app/Platform/Controllers/RelatedGamesTableController.php

```diff
@@ -54,23 +58,48 @@ public function __invoke(Request $request): View
             });
         }
 
+        if (isset($filterOptions['status'])) {
+            $this->gameListService->filterGameList(function ($game) use ($filterOptions) {
+                return $this->gameListService->useGameStatusFilter($game, $filterOptions['status']);
+            });
+        }
+
         $this->gameListService->mergeWantToPlay($loggedInUser);
 
         $this->gameListService->sortGameList($sortOrder);
 
         $availableSorts = $this->gameListService->getAvailableSorts();
-        $availableFilters = [
+        $availableCheckboxFilters = [
             'console' => 'Group by console',
             'populated' => 'Only with achievements',
         ];
+        $availableSelectFilters = [
+            [
+                'kind' => 'status',
+                'label' => 'Status',
+                'options' => [
+                    'all' => 'All games',
+                    'unstarted' => 'No achievements earned',
+                    'lt-beaten-softcore' => 'Has progress, but no award',
+                    'gte-beaten-softcore' => 'Beaten (softcore) or greater',
+                    'gte-beaten-hardcore' => 'Beaten or greater',
+                    'gte-completed' => 'Completed or mastered',
+                    'eq-mastered' => 'Mastered',
+                    'eq-beaten-softcore-or-beaten-hardcore' => 'Beaten, but still missing achievements',
+                    'any-softcore' => 'Has any softcore progress',
+                    'revised' => 'Completed or mastered, but the set was revised',
+                ],
+            ],
+        ];
```

Comment:
These should be hidden when not logged in (`$loggedInUser === null` or `$this->gameListService->userProgress === null`)

Reply:
Resolved in latest.